index.html

```diff
@@ -78,6 +64,76 @@ <h1 class="menu-title">Menu items</h1>
     <div class="container"></div>
   </main>
 
+  <!-- MODALS -->
+  <div class="modals">
+    <a href="#" id="modal-button" class="button add">Add</a>
+  </div>
+  <div id="addItemModal" class="modal">
+    <!-- Modal content -->
+    <div class="modal-content slideDown">
+      <div class="modal-header">
+        <span class="close closeAddModal" id="closeModal">&times;</span>
+        <h2>Add item</h2>
+      </div>
+      <div class="modal-body">
+        <form action="" class="modal-form">
+          <div class="form-row">
+            <label for="">Title</label>
```

Comment:
Consider providing the `id` of the element this label refers to, in this case it would be `title`.
This aids in accessibility and also enables functionality where clicking a label the referenced input is focused.